dietpi-install.sh

```diff
@@ -31,7 +100,11 @@ qm set "$ID" --memory "$RAM"
 qm set "$ID" --net0 'virtio,bridge=vmbr0'
 qm set "$ID" --scsi0 "$STORAGE:vm-$ID-disk-0"
 qm set "$ID" --boot order='scsi0'
-qm set "$ID" --scsihw virtio-scsi-pci
+qm set "$ID" --scsihw virtio-scsi-single
```

Comment:
What is actually the reason for this change? Is this a different VirtIO SCSI controller or some other option or did the identifier change with Proxmox versions?